invitations/tests/allauth/test_allauth.py

```diff
@@ -65,6 +65,38 @@ def test_accept_invite_allauth(self, method):
             email='email@example.com')
         self.assertTrue(allauth_email_obj.verified)
 
+    @parameterized.expand([
+        ('get'),
+        ('post'),
```

Comment:
I'm just trying to be consistent with the code style in the original repository.